yaksh/code_evaluator.py

```diff
@@ -108,9 +108,14 @@ def safe_evaluate(self, user_answer, test_case_data, file_paths=None):
 
         except TimeoutException:
             err = self.timeout_msg
+        except OSError:
+            msg = traceback.format_exc(limit=0)
+            err = "Error: {0}".format(msg)
         except Exception:
-            err = "Error: {0}".format(traceback.format_exc(limit=0))
-
+            exc_type, exc_value, exc_tb = sys.exc_info()
+            tb_list = traceback.format_exception(exc_type, exc_value, exc_tb)
+            del tb_list[1:3]
```

Comment:
I am not sure about this.  I don't think it is a good idea to do this and indiscriminately remove the two lines from `tb_list`.  What if tb_list has only two lines in it?  Can you pease ensure that this is also fully tested for different cases or write it a bit more safely.